bayes_people_tracker:
    filter_type: "UKF"                                  # The Kalman filter type: EKF = Extended Kalman Filter, UKF = Uncented Kalman Filter
    cv_noise_params:                                    # The noise for the constant velocity prediction model
        x: 1.4
        y: 1.4
        std_limit: 1.0                                  # upper limit for the standard deviation of the estimated position 
    detectors:                                          # Add detectors under this namespace
        object3d_detector:                              # Name of detector (used internally to identify them). Has to be unique.
            topic: "/object3d_detector_gpu/measurements" # The topic on which the people_msgs/PositionMeasurementArray is published
            observation_model: "CARTESIAN"              # Obeservation model: CARTESIAN or POLAR
            noise_params:                               # The noise for the cartesian or polar observation model
                x: 0.1
                y: 0.1
            matching_algorithm: "NN"                    # The algorthim to match different detections. NN = Nearest Neighbour, NNJPDA = NN Joint Probability Data Association
            seq_size: 4                                 # Minimum number of observations for new track creation
            seq_time: 0.3                               # Minimum interval between observations for new track creation
        yolo_detector:
            topic: "/rgbd_detection2d_3d/measurements"
            observation_model: "CARTESIAN"
            noise_params:
                x: 0.5
                y: 0.5
            matching_algorithm: "NN"
            seq_size: 5
            seq_time: 0.2
